boost-common/src/test/java/io/openliberty/boost/common/boosters/MPOpenTracingBoosterTest.java

```diff
@@ -0,0 +1,79 @@
+package io.openliberty.boost.common.boosters;
+
+import static io.openliberty.boost.common.config.ConfigConstants.*;
+import static io.openliberty.boost.common.utils.DOMUtils.getDirectChildrenByTag;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.RestoreSystemProperties;
+import org.junit.rules.TemporaryFolder;
+import org.w3c.dom.Element;
+
+import io.openliberty.boost.common.BoostException;
+import io.openliberty.boost.common.BoostLoggerI;
+import io.openliberty.boost.common.config.BoostProperties;
+import io.openliberty.boost.common.config.BoosterConfigurator;
+import io.openliberty.boost.common.config.LibertyServerConfigGenerator;
+import io.openliberty.boost.common.utils.BoostUtil;
+import io.openliberty.boost.common.utils.BoosterUtil;
+import io.openliberty.boost.common.utils.CommonLogger;
+import io.openliberty.boost.common.utils.ConfigFileUtils;
+
+public class MPOpenTracingBoosterTest {
+
+    @Rule
+    public TemporaryFolder outputDir = new TemporaryFolder();
+
+    @Rule
+    public final RestoreSystemProperties restoreSystemProperties = new RestoreSystemProperties();
+
+    BoostLoggerI logger = CommonLogger.getInstance();
+
+    /**
+     * Test that the mpHealth-1.0 feature is added as the default when the
+     * Microprofile target is set to 2.0 (1.6)
+     * 
+     * @throws ParserConfigurationException
+     * @throws TransformerException
+     * @throws IOException
+     * @throws BoostException
+     * @throws SecurityException
+     * @throws NoSuchMethodException
+     * @throws InvocationTargetException
+     * @throws IllegalArgumentException
+     * @throws IllegalAccessException
+     * @throws InstantiationException
+     */
```

Comment:
If we're not Javadoc-documenting any of these exceptions, maybe would be better to remove all these and make it shorter?